bin/ci/_includes.sh

```diff
@@ -122,10 +122,18 @@ allowed_failures=(
   "INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER"
 )
 if [[ " ${allowed_failures[*]} " =~ " ${ORCA_JOB} " ]]; then
-  set +e
+#  set +e
```

Comment:
There's no reason to keep commented out code. If it's not worth a comment explaining why it's there, it's not worth keeping.